src/main/pages/setup-kubernetes/kubernetes-multi-user.adoc

```diff
@@ -1,80 +1,87 @@
 * Install tiller on your cluster:
-** Create a https://github.com/kubernetes/helm/blob/master/docs/rbac.md[tiller serviceAccount]: `kubectl create serviceaccount tiller --namespace kube-system`
-** Bind it to the almighty cluster-admin role: `kubectl apply -f ./tiller-rbac.yaml`
-** Install tiller itself: `helm init --service-account tiller`
-* Ensure that you have an NGINX-based ingress controller. Note: This is the default ingress controller on MiniKube. You can start it with `minikube addons enable ingress`
-* DNS discovery should be enabled. Note: It is enabled by default in minikube.
+.. Create a https://github.com/kubernetes/helm/blob/master/docs/rbac.md[tiller serviceAccount]: `kubectl create serviceaccount tiller --namespace kube-system`.
+.. Bind it to the almighty cluster-admin role: `kubectl apply -f ./tiller-rbac.yaml`.
+.. Install tiller: `helm init --service-account tiller`.
+* Ensure that you have an NGINX-based Ingress controller. This is the default Ingress controller on Minikube. To start it use the `minikube addons enable ingress` command.
+* DNS discovery should be enabled. It is enabled by default in Minikube.
 
 [id="cluster-ip"]
-== Cluster IP
+== Obtaining the cluster IP
 
-* If your cluster is running on MiniKube, simply type `minikube ip` at your terminal
-* If your cluster is in the cloud, obtain the hostname or ip address from your cloud provider
+To obtain the cluster IP:
 
-In production, you should specify a hostname (see https://github.com/eclipse/che/issues/8694[here] why). In case you don’t have a hostname (e.g. during development), and would still want to use a host-based configuration, you can use services such as nip.io or xip.io.
+* If your cluster is running on MiniKube, type `minikube ip` at the terminal.
+* If your cluster is in the cloud, obtain the hostname or IP address from your cloud provider.
 
-In case you’re specifying a hostname, simply pass it as the value of the `ingressDomain` parameter below.
+In production, you must specify a hostname. For details see https://github.com/eclipse/che/issues/8694[Add an ability to use host-based routing instead of path-based for k8s infras]. If you do not have a hostname (example: during development) and want to use a host-based configuration, use services such as nip.io or xip.io.
 
-If you must use IP address (e.g. your corporate policy prevents you from using nip.io), you would also have to set `isHostBased` to `false`.
+If specifying a hostname, pass it as the value of the `ingressDomain` parameter in the deployment syntax in the following section.
+
+If using the IP address (example: your corporate policy prevents you from using nip.io), you must also set `isHostBased` to `false`.
 
 [id="deploy-syntax"]
-== Deploy Syntax
+== Deployment Syntax
 
-The context of the commands below is `che/deploy/kubernetes/helm/che`
+The context of the commands below is `che/deploy/kubernetes/helm/che`.
```

Comment:
Would this stand out more as an admonition?

Reply:
Robin, I did not incorporate this feedback because a section with just one admonition would make little sense. I'll not resolve this comment and see if you/others feel strongly about it and then decide what must be done with it :)